### PR TITLE
binary tree lab finished

### PR DESCRIPTION
**Pull request by lhchandler4**

### CODE REVIEW

**Comment by lhchandler4** [comment]
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code

File: challenges/src/main/java/Java/Tree/BinarySearchTree.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+package Java.Tree;
+
+public class BinarySearchTree<T extends Comparable<T>> {
+
+    TreeNode<T> rootNode;
+
+//    public void addTreeNodeRecur(T data){
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Tree/BinarySearchTree.java:7]: corpse code!
